src/app/public/modules/ui-config/ui-config.service.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  Injectable
+} from '@angular/core';
+
+import {
+  Observable
+} from 'rxjs';
+
+@Injectable()
+export class SkyUIConfigService {
+
+  public getConfig(key: string, defaultConfig?: any): Observable<any> {
```

Comment:
Curious, does `defaultConfig` need a type?

Reply:
`defaultConfig` can be anything the consumer wants to store as a setting, so it needs to be of type `any`.